onset_maproom/README.md

```diff
@@ -55,6 +55,13 @@ The widgets module contains a few functions of note:
 * `Number()`: This is a component for a number selector. The first argument is the HTML id,
    the second and third are the lower and upper bound respectively.
 
+# Docker Build Instructions
+
+To build the docker image, we have to use a work around so that pingrid.py will be included correctly, as
+docker doesn't normally allow files above the working directory in the hierarchy to be included
+
+    $ tar -czh . | sudo docker build -t <desired image name> -
```

Comment:
symlinking pingrid is somewhat problematic for building dockerfiles. If this workaround is fine, then we can just forge ahead as is

Reply:
Here's how we deal with it in fbfmaproom: https://bitbucket.org/iridl/x24-vector/src/ca47ae24bdbcdb2dbc5d6ba0a9e2f8e54a595936/fbfmaproom/Dockerfile#lines-40

Reply:
BTW symlinking pingrid is problematic for other reasons too. I had problems with it when setting up pytest. Let's discuss alternatives (but not on this PR).

Reply:
Has this been tested? I tried doing just that and Docker didn't actually resolve the symlink

Reply:
Ah, you're right. Yes the fbfmaproom Dockerfile works, but it's written to expect the parent directory as context. That information is embedded in the release script. Your way is better, because it won't force a rebuild when the only changes are in other applications that share the repo.

As I said elsewhere, you need to add a release script, and we'll discuss that on our next call.